components/utils/urls.js

```diff
@@ -0,0 +1,2 @@
+
+export const URL = "https://api.quotable.io/random"
```

Comment:
I would personally declare this one url in the same file as where it is used in this case but I understand that in most cases, we would want to hide our api keys and by having it in a separate file, we can add the file in the .gitignore.